src/components/exercises/SyllableTable.js

```diff
@@ -8,56 +8,76 @@ import { RoundTextButton, RoundText } from '../Components';
 
 const mapIndexed = addIndex(map);
 
-const SyllableTable = ({ syllables, sound, index }) => {
+const SyllableTable = (
+  { letters, vowels, sound, indices: { letterIndex, vowelIndex } }
+) => {
+  let key = 0;
+
   const play = () => {
     sound.playAsync();
     sound.setPlaybackFinishedCallback(() => {
       sound.setPositionAsync(0);
     });
   };
 
-  return (
-    <View
-      style={{
-        flex: 1,
-        backgroundColor: PRIMARY,
-        alignItems: 'center'
-      }}
-    >
+  const createSyllables = (letter, letterKey) => {
```

Comment:
I think it makes sense to move that to its own component because
* currently `createSyllables` will be a new function on every render (minor issue).
* the signature is basically a React component.

So how about:

```.js
const Syllables = ({ letter, letterKey /* what ever props you additionally need */ }) => {
  // createSyllables
}

const SyllableTable = (/* props */) => {
  // ...

  return (
    <View
      style={{
      flex: 1,
      backgroundColor: PRIMARY,
      alignItems: 'center'
    }}
    >
    <View
      style={{
        alignItems: 'center',
        justifyContent: 'center',
        flexDirection: 'column',
        marginTop: 50
      }}
    >
    {mapIndexed(
      (letter, letterKey) => <Syllables key={letterKey} letter={letter} letterKey={letterKey} {...props} />
    )}
    </View>
  </View>
}
```